libkbfs/md_ops_test.go

```diff
@@ -397,12 +422,7 @@ func testMDOpsGetForHandlePublicFailVerify(t *testing.T, ver MetadataVer) {
 	h := parseTlfHandleOrBust(t, config, "alice,bob", true)
 	rmds, _ := newRMDS(t, config, h)
 
-	// Do this before setting tlfHandle to nil.
-	expectedErr := libkb.VerificationError{}
-	verifyMDForPublic(config, rmds, expectedErr, nil)
-
-	config.SetCrypto(failVerifyCrypto{config.Crypto(), expectedErr})
-
+	rmds.MD.(MutableBareRootMetadata).SetRefBytes(100)
```

Comment:
It's hard to understand why this causes a `MDMismatchError`.  I guess it changes the MdID?  Can you add a comment maybe?